### PR TITLE
[Fixtures] Allow no shipping and payments in fixtures

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A
| --------------- | -----
| Branch?         | 1.4
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | replaces https://github.com/Sylius/Sylius/pull/10503, fixes https://github.com/Sylius/Sylius/issues/10502, https://github.com/Sylius/Sylius/issues/10490
| License         | MIT

Thank you @igormukhingmailcom for your work! It's rebased and refactored now

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Fixture/OrderFixture.php [hunk 1]
```diff
@@ -193,6 +197,7 @@ private function generateItems(OrderInterface $order): void
                 /** @var OrderItemInterface $item */
                 $item = $generatedItems[$variant->getCode()];
                 $this->orderItemQuantityModifier->modify($item, $item->getQuantity() + random_int(1, 5));
+                $variant->setShippingRequired($shippingMethodsAvailable);
```
Comment (lchrusciel) [inline src/Sylius/Bundle/CoreBundle/Fixture/OrderFixture.php]: I think it should be an option for product fixtures and we shouldn't change previous items in other fixtures.

Comment (pamil) [reply src/Sylius/Bundle/CoreBundle/Fixture/OrderFixture.php]: Yeah, we shouldn't change variants in order fixture.

File: src/Sylius/Bundle/CoreBundle/Fixture/OrderFixture.php [hunk 2]
```diff
@@ -238,6 +234,14 @@ private function selectShipping(OrderInterface $order): void
             return;
         }
 
+        $channel = $order->getChannel();
+        if (count($this->shippingMethodRepository->findEnabledForChannel($channel)) === 0) {
```
Comment (pamil) [inline src/Sylius/Bundle/CoreBundle/Fixture/OrderFixture.php]: This repository call is doubled below.

**Comment by pamil** [comment]
Thank you, Mateusz! :1st_place_medal:

**Comment by Zales0123** [comment]
And Igor! 🎉

**Pull request merged**